hoth_api.yaml

```diff
@@ -568,6 +573,103 @@ components:
           example: "1st QTR FY23"
         order:
           type: integer
+    RequirementsChecklist:
+      description: >-
+        Schema used to generate Requirements Checklist document
+      type: object
+      properties:
+        projectOverview:
+          type: object
+          properties:
+            title:
+              type: string
+            scope:
+              type: string
+            emergencyDeclaration:
+              type: boolean
+        organization:
+          type: object
+          properties:
+            agency:
+              type: string
+            name:
+              description: >-
+                The DISA Organization or Organization Name column
+              type: string
+        contacts:
+          type: object
+          properties:
+            missionOwnerName:
+              type: string
+            cor:
+              $ref: "#/components/schemas/CorContact"
+            acor:
+              $ref: "#/components/schemas/CorContact"
+        currentContract:
+          type: object
+          properties:
+            exists:
+              type: boolean
+            incumbentContractorName:
+              type: string
+            contractNumber:
+              type: string
+            taskOrderNumber:
+              type: string
+            contractExpiration:
+              type: string
+        exceptionTofairOpportunity:
+          type: string
+          enum:
+            - NO_NONE
+            - YES_FAR_16_505_B_2_I_A
+            - YES_FAR_16_505_B_2_I_B
+            - YES_FAR_16_505_B_2_I_C
+        periodOfPerformance:
+          $ref: "#/components/schemas/PeriodOfPerformance"
+        contractType:
+          type: object
+          properties:
+            firmFixedPrice:
+              type: boolean
+            timeAndMaterials:
+              type: boolean
+            justification:
+              type: string
+        sensitiveInformation:
+          type: object
+          properties:
+            baaRequired:
+              type: boolean
+            potentialToBeHarmful:
+              type: boolean
+            foiaContact:
+              type: object
+              properties:
+                fullName:
+                  type: string
+                email:
+                  type: string
+                address:
+                  type: string
+            section508:
+              type: object
+              properties:
+                section508Sufficient:
+                  type: boolean
+    CorContact:
+      description: >-
+        A Task Order (Alternative) Contracting Officer Representative (COR/ACOR) information
+      type: object
+      properties:
+        name:
+          type: string
+        email:
+          type: string
+        phoneNumber:
+          type: string
+        dodaa:
```

Comment:
Department of Defense Activity Address Code (DoDAAC) ?

Reply:
```suggestion
        dodaac:
```

Reply:
Added as `description`.

Reply:
I think the property name needs a `c` at the end.  I added the code suggestion later, you may not have seen it.